src/js/EditController.js

```diff
@@ -35,9 +35,20 @@ goog.require('ga_mvt_service');
       $scope.layer = layer;
       $scope.globals.isEditActive = toggle;
       $scope.globals.pulldownShown = toggle;
-      if (toggle) {
-        gaMvt.reload($scope.layer);
+    });
+
+    var activate = function(active) {
+      if (active && $scope.layer && $scope.layer.glStyle) {
+        // set timeout to smooth the transition
+        setTimeout(function() {
+          gaMapUtils.applyGlStyleToOlLayer(
+              $scope.layer, $scope.layer.glStyle)
+        }, 600);
       }
+    };
+
+    $scope.$watch('globals.isEditActive', function(active) {
+      activate(active);
```

Comment:
you can 't reload the style, only here. For example, if someone click on the 'open advanced link' in the feedback directive and the advancedStyle is already open , the style will not be reloaded. You also need to activate it in gaToggleEdit as before.

Reply:
>  For example, if someone click on the 'open advanced link' in the feedback directive and the advancedStyle is already open , the style will not be reloaded. 

And this is correct. I was correcting the error where clicking the menu "Style the map" wouldn't re-apply the style. Anyway since we move all the edit functions to the advance mode and all editions will now be permanent we won't have to handle 2 styles anymore.